model/client4.go

```diff
@@ -1186,6 +1186,17 @@ func (c *Client4) UpdateUserPassword(userId, currentPassword, newPassword string
 	return CheckStatusOK(r), BuildResponse(r)
 }
 
+// UpdateUserHashedPassword updates a user's password with an already-hashed password. Must be a system administrator.
+func (c *Client4) UpdateUserHashedPassword(userId, newHashedPassword string) (bool, *Response) {
+	requestBody := map[string]string{"already_hashed": "true", "new_password": newHashedPassword}
```

Comment:
Here we need to include an empty `current_password` for the endpoint not to complain
```suggestion
	requestBody := map[string]string{"already_hashed": "true", "new_password": newHashedPassword, "current_password": ""}
```

Reply:
Said nothing, I was making the wrong request... :man_facepalming: